gnat2goto/driver/tree_walk.adb

```diff
@@ -1639,12 +1684,14 @@ package body Tree_Walk is
                   Cond : Irep;
                   Post : Irep;
 
-                  Bound_Low  : constant Irep :=
-                    Do_Expression (Low_Bound (Dsd));
-                  Bound_High : constant Irep :=
-                    Do_Expression (High_Bound (Dsd));
+                  Bound_Low  : Irep;
+                  Bound_High : Irep;
 
                begin
+                  Bound_Low := Do_Expression (Low_Bound (Dsd));
+                  Bound_High :=
+                     Do_Expression (High_Bound (Dsd));
```

Comment:
This was done for debugging purposes, can you please move this back? FYI gdb understands Ada code, you can do a `catch exception` to break on an ada exception.